### ISSUE TITLE
get_translations is not working

### ISSUE THREAD

**Issue by MShafquat** [KartikTalwar/Duolingo#75]
When I am passing a list with source and target it shows:
```
Traceback (most recent call last):
  File "/home/shafquat/DuolingoWordList/venv/lib/python3.7/site-packages/duolingo.py", line 230, in get_translations
    return request.json()
  File "/home/shafquat/DuolingoWordList/venv/lib/python3.7/site-packages/requests/models.py", line 897, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "get_vocab.py", line 15, in <module>
    meaning = lingo.get_translations(wordstrings, source='ar', target='en')
  File "/home/shafquat/DuolingoWordList/venv/lib/python3.7/site-packages/duolingo.py", line 232, in get_translations
    raise Exception('Could not get translations')
Exception: Could not get translations
```

**Comment by joshcoales** [comment]
It's still working on my branch [which is in PR]: https://travis-ci.org/joshcoales/Duolingo/builds/639460517
Have you provided both a username and password to the Duolingo constructor? It no longer works with just username

**Comment by MShafquat** [comment]
Yeah, I provided both username and password in the constructor.

**Comment by joshcoales** [comment]
Very odd..
If I try this as an example:
```
response = self.lingo.get_translations(['a', 'e'], source='ar', target='en')
```
it works fine, and returns a response I would expect:
```
{'a': ['-'], 'e': []}
```

Could you say what's in your `wordstrings` or perhaps debug and see what the request response is?

**Comment by mrichar1** [comment]
I've just hit this - it looks like it's triggered by length of word list. E.g. I have a word list containing my whole vocab (around 1000 words) - if I slice the list into 2 halves its fine, but the whole list errors:

```
lingo.get_translations(words[:500]) # works fine
lingo.get_translations(words[500:]) # works fine
lingo.get_translations(words) # error
```

I tried a bit of interval halving, and found that 903 is the cut off - which seems arbitrary enough that I suspect it's down to something else like the total character length of the list or something?

**Comment by joshcoales** [comment]
Ahh, that is interesting. Could well be the character length, perhaps there's a limit to how long the query string can be. We could automatically slice it up, how long in total characters were the 903 words?

**Comment by mrichar1** [comment]
So at 903 the total length (`json.dumps(wordlist))` was 11940. so I made lists of varying lengths and found:

```
wordlist = ['aaaaaaaa'] * 1066 # length is 12792  OK
wordlist = ['aaaaaaaa'] * 1067 # length is 12804 FAIL
```
So the cut-off looks to be 12800 which seems more sensible as a ^2.

However there is another threshold:

```
wordlist = ['a'] * 1999 # len 9995 OK
wordlist = ['a'] * 2000` # len is 10000 FAIL
```

So it looks like there are 2 rules - no list with total length > 1280, and no list with more than 1999 elements. I'd be interested to know if you see the same resultS

**Comment by joshcoales** [comment]
Seems odd that the json length is based on json.dumps(), not json.dumps(*, separators=(',', ':'))
But maybe something internal to duolingo.
Should be fixed in this PR though, I'll get that merged and published once tests run through

**Comment by joshcoales** [comment]
Fixed in 0.5.3
